### DISCUSSION TITLE
Not working

### DISCUSSION THREAD

**Discussion by net-frog** [aoout/obsidian-epub-importer#46] [General]
I am not sure if I am doing it right but after running the import and selecting the epub file nothing happens. Please share the steps to import epub.

**Comment by aoout** [comment]
show your settings first.

**Comment by net-frog** [comment]
Please find it below:

{
  "tag": "book",
  "libraries": [
    "C:\\Users\\user1\\Documents\\Personal\\ObsidianVault\\EPUBs"
  ],
  "byDrag": false,
  "savePath": "",
  "assetsPath": "{{savePath}}/{{bookName}}/images",
  "granularity": 4,
  "mocName": "{{bookName}}",
  "notePropertysTemplate": "",
  "oneNote": false,
  "oneFolder": true,
  "mocPropertysTemplate": "title: {{bookName}}\nauthor: {{author}}\npublisher: {{publisher}}\nstatus: false",
  "imageFormat": "![](imagePath)",
  "autoOpenRightPanel": false,
  "allbooks": false,
  "removeDuplicateFolders": false,
  "moreLog": true
}

**Comment by net-frog** [comment]
Just to add I have checked in the developer console and it is throwing "failed to parse .epub file"

**Reply by Hosseingsd** [reply]
The "Save Path" field should contain a relative path within the obsidian library, not an absolute path.

**Comment by aoout** [comment]
Please create an issue and upload the .epub file.
